data/processors/coords.py

```diff
@@ -119,8 +122,31 @@ def check_coords(input_data):
             )
 
 
+def validate_coords(input_data):
```

Comment:
Added a method docstring
```suggestion
def validate_coords(input_data):
    """Check that coordinates are not too far away from their parent"""
```